Stellar-contract-meta.x

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+// The contract spec XDR is highly experimental, incomplete, and still being
+// iterated on. Breaking changes expected.
+
+% #include "xdr/Stellar-types.h"
+% #include "xdr/Stellar-contract.h"
+namespace stellar
+{
+
+struct SCMetaCustom
+{
+    opaque key<64>;
+    opaque value<64>;
+};
+
+struct SCMetaVersionSemVer
```

Comment:
Wouldn't you need to include the language here? And what if someone is using a different (custom) build of the sdk?

Reply:
Or is this just to scope and switch the host functions?

Reply:
Ahh good point, there is not necessarily just one SDK in the future. We could make that different enum values?